### ISSUE TITLE
what will happen if I modify some config

### ISSUE THREAD

**Issue by passion3394** [MaybeShewill-CV/lanenet-lane-detection#97]
In config/global_config.py, I find the following config

# Set the shadownet validation batch size
__C.TRAIN.VAL_BATCH_SIZE = 8
# Set the learning rate decay steps
__C.TRAIN.LR_DECAY_STEPS = 410000
# Set the learning rate decay rate
__C.TRAIN.LR_DECAY_RATE = 0.1
# Set the class numbers
__C.TRAIN.CLASSES_NUMS = 2
# Set the image height
__C.TRAIN.IMG_HEIGHT = 256
# Set the image width
__C.TRAIN.IMG_WIDTH = 512

what will happen if I change the IMG_HEIGHT and IMG_WIDTH to a bigger value? thanks.

**Comment by MaybeShewill-CV** [comment]
@passion3394 That means the model will cost more gpu memory so you may reduce the batch size according to your gpu device:)

**Comment by passion3394** [comment]
@MaybeShewill-CV what's the criteria of the modifying? the IMG_HEIGHT and the IMG_WIDTH should be divided by 16 or 32?

**Comment by MaybeShewill-CV** [comment]
@passion3394 That may depends on the experiment I have not tested it before:)

**Comment by passion3394** [comment]
theoretically, if I modify the  IMG_HEIGHT and the IMG_WIDTH to a higher value, the precision will higher? because we use more features in one batch.

**Comment by MaybeShewill-CV** [comment]
@passion3394 The precision and the size of input tensor are not that related:)

**Comment by MaybeShewill-CV** [comment]
@passion3394 That depends on the data that fed into the model and the data that you want to process with:)

**Comment by passion3394** [comment]
@MaybeShewill-CV I use my own dataset, and train the model about 20000 steps, but the training loss is always about 1.2, and the training accuracy is about 75%, the learning rate multiplys by 0.9 every 1000 step. Can you give me some advise about the training process?

**Comment by passion3394** [comment]
@MaybeShewill-CV the dense net is better than vgg?

**Comment by MaybeShewill-CV** [comment]
@passion3394 You may test it:)

**Comment by passion3394** [comment]
@MaybeShewill-CV Hi, I have two question, could you tell me something about them.

(1)The paper says that they do the model on the 512 * 256 size images, what if I modify the model and apply to other size of images?  Can this way work on the lanenet-lane-detection code?

(2)I want to get the coordinates of the detected lanes, and I have dug into the code, saw these codes:


def _get_lane_area(binary_seg_ret, instance_seg_ret):
        """
        通过二值分割掩码图在实例分割图上获取所有车道线的特征向量
        :param binary_seg_ret:
        :param instance_seg_ret:
        :return:
        """
        idx = np.where(binary_seg_ret == 1)

        lane_embedding_feats = []
        lane_coordinate = []
        for i in range(len(idx[0])):
            lane_embedding_feats.append(instance_seg_ret[idx[0][i], idx[1][i]])
            lane_coordinate.append([idx[0][i], idx[1][i]])

        return np.array(lane_embedding_feats, np.float32), np.array(lane_coordinate, np.int64)

the coordinates are computed through the resized 512*256 image?

**Comment by MaybeShewill-CV** [comment]
@passion3394 1.The model can be feed into a input tensor with different size. 2.The coord are computed through the resized 512*256 image

**Comment by passion3394** [comment]
@MaybeShewill-CV ok,thanks, I just want to try if I modify the 512*256 to other size of the network,  will the network get a better precision? especially on custom datasets.

**Comment by MaybeShewill-CV** [comment]
@passion3394 You may test it:)

**Comment by passion3394** [comment]
@MaybeShewill-CV ok, thanks,will close this issue.

**Comment by passion3394** [comment]
@MaybeShewill-CV how to calculate the accuracy of lanenet? is that similar with the IoU in the semantic segmentation?

**Comment by MaybeShewill-CV** [comment]
@passion3394 You may refer to this #101 to see if it can help you:)

**Comment by passion3394** [comment]
@MaybeShewill-CV thanks, it helps me a lot.

**Comment by MaybeShewill-CV** [comment]
@passion3394 Welcome:)
